boards/arm/arduino_nicla_sense_me/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Arduino Nicla Sense ME board configuration
```

Comment:
This file is optional, so as it is anyway empty it can simply be removed.

Reply:
Yes, thank you 👍